irohad/consensus/yac/storage/impl/yac_common.cpp

```diff
@@ -33,15 +33,15 @@ namespace iroha {
         auto first = votes.at(0);
         return std::all_of(
             votes.begin(), votes.end(), [&first](const auto &current) {
-              return first.hash.proposal_hash == current.hash.proposal_hash;
+              return first.hash.vote_round == current.hash.vote_round;
             });
       }
 
       boost::optional<ProposalHash> getProposalHash(
```

Comment:
This method is used only in tests now, and previously it was testing that round is the same. Please remove the method, or change its behavior to return the round.